code/modules/gear_presets/_select_equipment.dm

```diff
@@ -774,6 +774,49 @@ var/list/rebel_rifles = list(
 			H.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/knife(H), WEAR_FEET)
 			H.equip_to_slot_or_del(new /obj/item/storage/firstaid/toxin(H.back), WEAR_IN_BACK)
 			H.equip_to_slot_or_del(new /obj/item/device/motiondetector(H.back), WEAR_IN_BACK)
+		if(11) //PMC support synth
+			H.equip_to_slot_or_del(new /obj/item/card/id/pmc(H), WEAR_ID)
+			H.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/PMC, WEAR_L_EAR)
+			H.equip_to_slot_or_del(new /obj/item/clothing/under/marine/veteran/PMC, WEAR_BODY)
+			H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/marine/veteran/PMC, WEAR_JACKET)
+			H.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine/veteran/PMC, WEAR_HANDS)
+			H.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/marine/veteran/PMC, WEAR_HEAD)
+			H.equip_to_slot_or_del(new /obj/item/clothing/shoes/veteran/PMC/knife, WEAR_FEET)
+			H.equip_to_slot_or_del(new /obj/item/clothing/mask/rebreather/scarf, WEAR_FACE)
+			H.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel, WEAR_BACK)
+			H.equip_to_slot_or_del(new /obj/item/storage/belt/medical/full, WEAR_WAIST)
+			H.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health, WEAR_EYES)
+			faction = FACTION_PMC
```

Comment:
this change's the datum's faction, not the synthvivor. if this does properly work on the synthvivor (VV to confirm), then you'll just need to reset the faction to its initial value at the start of this proc so the other synthvivors (futureproofing) doesn't get the wrong faction

Reply:
Yeah I set the faction to the default survivor faction group on the next case immediately after to fix this

Reply:
wuh